### DISCUSSION TITLE
Why color is difference

### DISCUSSION THREAD

**Discussion by man44587** [google/model-viewer#4579] [Q&A]
Hi 
 
I want to set model same as https://modelviewer.dev/editor/ bit I dont why color is difference

from https://modelviewer.dev/editor/ I can see grey line perfectly
![image](https://github.com/google/model-viewer/assets/44811963/6bb42436-b6d5-496e-a30f-3f297a5781b7)

from mine grey line not show
![image](https://github.com/google/model-viewer/assets/44811963/2e0d337f-3527-4acd-b9c6-2e296a2a65fd)

I already set same component same hdr, exposure, alphafactor

What did I miss? to make same as https://modelviewer.dev/editor/

**Comment by elalish** [comment]
My best guess is you're using a different version of MV, which is common because our editor runs off our latest master branch, while you're likely using a release version. Try testing `https://modelviewer.dev/node_modules/@google/model-viewer/dist/model-viewer.min.js` to see if that's consistent. Don't use this in production! If that works, then our next release should fix the problem for you.
